Zend/zend_compile.h

```diff
@@ -1001,9 +1008,9 @@ ZEND_API zend_string *zend_type_to_string(zend_type type);
 #define ZEND_FETCH_CLASS_ALLOW_UNLINKED 0x0400
 #define ZEND_FETCH_CLASS_ALLOW_NEARLY_LINKED 0x0800
 
-/* These should not clash with ZEND_ACC_(PUBLIC|PROTECTED|PRIVATE) */
-#define ZEND_PARAM_REF      (1<<3)
-#define ZEND_PARAM_VARIADIC (1<<4)
+/* These should not clash with ZEND_ACC_PPP_MASK and ZEND_ACC_PPP_SET_MASK */
+#define ZEND_PARAM_REF      (1<<13)
+#define ZEND_PARAM_VARIADIC (1<<14)
```

Comment:
I'm confused, was there any need to renumber them? They still don't clash?

Reply:
These flags are used for params, and so can the property flags (constructor property promotion).

Reply:
Ah, I see what you mean now. I can't check this on mobile, I'll do so when I'm home.